spec-proposal/scalars-implementation-guide.md

```diff
@@ -0,0 +1,112 @@
+# GraphQL Scalars implementation guide
+
+_Note: This proposal is to be discussed at a GraphQL Working Group meeting. We would like to clarify scalar implementations, particularly to make custom scalars easier to implement._
+
+Scalar types are primitive types in the GraphQL type system, usable for input and output.
+
+## Serialization and Deserialization
+A GraphQL request often requires a deserialization step before being executed and and a serialization of the execution result. By far the most important example is a GraphQL request send over HTTP POST via JSON. The request body is a JSON document that needs to be deserialized containing the GraphQL document and optionally an operation name and variables.
```

Comment:
```suggestion
A GraphQL request often requires a deserialization step before being executed and a serialization of the execution result. By far the most important example is a GraphQL request send over HTTP POST via JSON. The request body is a JSON document that needs to be deserialized containing the GraphQL document and optionally an operation name and variables.
```

Reply:
Thanks

FYI to explain why this PR isn't active, we are continuing the work in https://github.com/graphql/graphql-scalars

Reply:
ok